RandLAPACK/drivers/rl_hqrrp.hh

```diff
@@ -598,12 +600,14 @@ int64_t NoFLA_QRPmod_WY_unb_var4(
 //    The typical use-case for this function is to call with build_T=true.
 //    Calling with build_T=false is only done at HQRRP's last iteration.
 //
-
+/*-----------------------------------COMMENTED THIS OUT TO TEST THE BASIC HQRRP-----------------------------------*/
+/*
     if (!pivoting && !use_cholqr) {
         return GEQRF_mod_WY(num_stages, m_A, n_A, buff_A, ldim_A, buff_t, buff_T, ldim_T);
     } else if (!pivoting && use_cholqr) {
         return CHOLQR_mod_WY(num_stages, m_A, n_A, buff_A, ldim_A, buff_t, buff_T, ldim_T, buff_R, ldim_R, buff_D);
     }
+*/
```

Comment:
Please make a decision on whether this code should be here. If you want it commented out then make sure to comment-out variables that are only used here (I suspect that use_cholqr is only used here).

Reply:
Same comment as with CQRRP CPU